src/components/course/CoursePage.jsx

```diff
@@ -198,6 +198,7 @@ const CoursePage = () => {
     courseEntitlements: courseState?.course?.entitlements,
     activeCourseRun: courseState?.activeCourseRun,
     userSubsidyApplicableToCourse,
+    canEnrollListPrice: subsidyAccessPolicyRedeemabilityData?.coursePrice,
```

Comment:
[suggestion] Given both the `courseListPrice` function argument above to `useUserSubsidyApplicableToCourse` and `useCoursePriceForUserSubsidy` here rely on:
```
courseState?.activeCourseRun?.firstEnrollablePaidSeatPrice || getEntitlementPrice(courseState?.course?.entitlements))
```
...it may make sense to calculate this once here in `CoursePage` and pass a singular `courseListPrice` around where needed to avoid needing to have the same logic in multiple places.

Reply:
I agree.  Done!